src/Neo.Cryptography.BLS12_381/Bls12.Adder.cs

```diff
@@ -29,10 +29,10 @@ Fp12 IMillerLoopDriver<Fp12>.AdditionStep(in Fp12 f)
             return Ell(in f, in coeffs, in P);
         }
 
-        static Fp12 IMillerLoopDriver<Fp12>.SquareOutput(in Fp12 f) => f.Square();
+        public static Fp12 SquareOutput(in Fp12 f) => f.Square();
 
-        static Fp12 IMillerLoopDriver<Fp12>.Conjugate(in Fp12 f) => f.Conjugate();
+        public static Fp12 Conjugate(in Fp12 f) => f.Conjugate();
 
-        static Fp12 IMillerLoopDriver<Fp12>.One => Fp12.One;
+        public static Fp12 One => Fp12.One;
```

Comment:
Dotnet core only accept public?

Reply:
what do you mean? If you asking the reason why i put `public`, is because it so it `public`. That's what it was on the interface.